kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-core/src/main/resources/org/kie/server/services/taskassigning/solver/taskAssigningDefaultSolverConfig.xml

```diff
@@ -11,4 +11,8 @@
     <scoreDrl>org/kie/server/services/taskassigning/solver/taskAssigningScoreRules.drl</scoreDrl>
   </scoreDirectorFactory>
 
+  <termination>
+    <unimprovedSpentLimit>PT2M</unimprovedSpentLimit>
```

Comment:
How did you come up with this particular limit? Is there any rationale?

Reply:
From conversation with @ge0ffrey, let's say that it's a kind of suggested value.